polars/polars-io/src/csv.rs

```diff
@@ -623,27 +623,27 @@ hello,","," ",world,"!""#;
     #[test]
     fn test_very_long_utf8() {
         let csv = r#"column_1,column_2,column_3
--86.64408227,"Lorem Ipsum is simply dummy text of the printing and typesetting 
```

Comment:
My editor removed this trailing whitespace by default, is it needed?

Reply:
I think its ok.